<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleSettingsWidget</class>
 <widget class="QWidget" name="ConsoleSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Region:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="region">
       <item>
        <property name="text">
         <string>Auto-Detect</string>
        </property>
       </item>
       <item>
         <property name="text">
           <string>NTSC-J (Japan)</string>
         </property>
       </item>
       <item>
        <property name="text">
         <string>NTSC-U (US)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PAL (Europe, Australia)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BIOS Path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="biosPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="biosPathBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="enableTTYOutput">
       <property name="text">
        <string>Enable TTY Output</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="fastBoot">
       <property name="text">
        <string>Fast Boot</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="enableSpeedLimiter">
       <property name="text">
        <string>Enable Speed Limiter</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="pauseOnStart">
       <property name="text">
        <string>Pause On Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
